testplan/report/testing/base.py

```diff
@@ -509,6 +510,8 @@ def __init__(self,
         self.part = part  # i.e. (m, n), while 0 <= m < n and n > 1
         self.part_report_lookup = {}
 
+        self.extra_attributes = extra_attributes
```

Comment:
What exactly is the purpose of storing these extra attributes?

Reply:
For now, this is needed by internally used RFBEmailExporter. I think we can record these attributes (if nothing defined it's None) in test report, maybe useful in the future. But I guess maybe we need to check this content should be jsonified.

Reply:
ok thanks for clarifying